.github/workflows/release.yml

```
@@ -249,3 +249,53 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG: ${{ steps.create-release.outputs.computed-prefix }}${{ steps.create-release.outputs.version }}
+
+  check-plugin-version:
```

Nit: 
```suggestion
  upload-plugin-version:
```
would be more accurate